light-rpc/README.md

```diff
@@ -0,0 +1,26 @@
+# Light RPC
+
+Submitting a [transaction](https://docs.solana.com/terminology#transaction) to be executed on the solana blockchain,
+requires the client to identify the next few leaders based on the
+[leader schedule](https://en.wikipedia.org/wiki/QUIC), look up their peering information in gossip and
```

Comment:
Leader shedule is pointing to QUIC

Reply:
ahh fixing

Reply:
fixed in [41a1a61](https://github.com/blockworks-foundation/solana/pull/7/commits/41a1a61a561420d6d32241b65e3a14150d3decdd)